test/lib/jdk/test/lib/Asserts.java

```diff
@@ -269,7 +269,7 @@ public static void assertEqualsByteArray(byte[] lhs, byte[] rhs, String msg) {
         if (!Arrays.equals(lhs, rhs)) {
             msg = Objects.toString(msg, "assertEqualsByteArray")
                     + ": expected " + HexFormat.of().formatHex(lhs)
-                    + " to equal " + HexFormat.of().formatHex(rhs);
+                    + " but was " + HexFormat.of().formatHex(rhs);
```

Comment:
The original message is about these two bytes being unequal, but with the new message, it seems to imply that the first argument `lhs` is the expected value and `rhs` is the actual value. I grokked a few calls of this method and it seems that this may not always be the case?

Reply:
Are those calls from me? I know I haven't followed this pattern and I'm thinking about fixing them later.

The reason I want to make this change is to make it consistent with the current `assertEquals` method that shows " expected: LEFT but was: RIGHT". There are quite a lot of calls like `assertEquals(variable, "literal")`, but I think that's the callers' problem instead of the method's.

Reply:
Could you rename `lhs` and `rhs` to `expected` and `actual`? Just to avoid the problems mentioned in https://github.com/openjdk/jdk/pull/17952#issuecomment-1973131407

Reply:
I added a new commit with the new names. Only methods I touched in this PR were updated.